OutOfSchool/OutOfSchool.EmailSender/ServiceProviderExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace OutOfSchool.EmailSender
+{
+    public static class ServiceProviderExtensions
+    {
+        public static IServiceCollection AddEmailSender(
+            this IServiceCollection services,
+            Action<OptionsBuilder<EmailOptions>> emailOptions,
+            Action<OptionsBuilder<SmtpOptions>> smtpOptions)
+        {
+            services.AddSingleton<IEmailSender, EmailSender>();
+            if (emailOptions == null)
+            {
+                throw new ArgumentNullException(nameof(emailOptions), @"Please provide emailOptions for EmailSender");
+            }
```

Comment:
strings can be moved to resources

Reply:
removed second parameter because it repeats standard message